### PR TITLE
Stranded mode

### PR DESCRIPTION
**Pull request by Shiiyu**
image
image
image

### CODE REVIEW

File: views/stats.ejs [hunk 1]
```diff
@@ -1140,11 +1150,15 @@ const metaDescription = getMetaDescription()
           <% } %>
           <% if(calculated.profile.game_mode == 'ironman'){ %>
             <% if(notAvailable.length > 0){ %><br><br><% } %>
-            This is an <strong>Ironman</strong> profile. The player cannot use the auction house, bazaar, trade or pick up drops from other players.
+            This is an <strong>Ironman</strong> profile. The player cannot use the auction house, bazaar, trade, or pick up drops from other players.
```
[the remark this answers is not in the document]
Comment (Shiiyu) [reply views/stats.ejs:1153]: Yes, it needed an Oxford comma.

File: views/stats.ejs [hunk 2]
```diff
@@ -1140,11 +1150,15 @@ const metaDescription = getMetaDescription()
...
           <% } %>
           <% if(calculated.profile.game_mode == 'bingo'){ %>
             <% if(notAvailable.length > 0){ %><br><br><% } %>
-            This is a <strong>Bingo</strong> profile. The player cannot spend gems, use the auction house, bazaar, trade or pick up drops from other players.
+            This is a <strong>Bingo</strong> profile. The player cannot spend gems, use the auction house, bazaar, trade, or pick up drops from other players.
+          <% } %>
+          <% if(calculated.profile.game_mode == 'island'){ %>
+            <% if(notAvailable.length > 0){ %><br><br><% } %>
+            This is a <strong>Stranded</strong> profile. The player cannot leave their skyblock island or trade with other players.
```
Comment (metalcupcake5) [inline views/stats.ejs:1161]: is trading with other players really needed

Comment (dukio) [reply views/stats.ejs:1161]: I had the same though but then realised people can still /visit stranded gamemode players so... I guess it's good to specify that

Comment (Shiiyu) [reply views/stats.ejs:1161]: I think so because other people can still visit their island

**Review by metalcupcake5 — APPROVED** [review]
lgtm didnt test

**Pull request merged**